ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/plugin/DependencyPropertiesDialog.java

```diff
@@ -61,16 +62,17 @@ public DependencyPropertiesDialog(boolean editable, ExportPackageObject object)
 
 	public DependencyPropertiesDialog(boolean editable, boolean showReexport, boolean export, boolean optional, String version, boolean showOptional, boolean isImport, String pluginId) {
 		super(PDEPlugin.getActiveWorkbenchShell());
+		isPlugin = showReexport;
```

Comment:
If `isPlugin` equals `fShowReexport` it should not need a new field.

Beside this "fShowReexport" seem to indicate it is more about reexported packages?